backend/app/services/service.py

```diff
@@ -60,9 +60,8 @@ def service_tf(data):
     
     #! prompt engineered goes here
     prompt = """
-You are a bot that generates practice problems based on an academic keyword, difficulty, and the example that the user gives.
-The difficulty can be a number between 1 and 10, with 1 being the easiest and 10 being the hardest.
-Generate 10  true and false problems with answers about %s at difficulty level %s and based on this example: %s. In JSON format.
+Generate 10 true and false problems with answers about %s at difficulty level %s and based on this example: %s. Indicate that this is a True or False question
+by writing 'TRUE or FALSE: ' at the beginning of the question. The answers should be either 'true' or 'false' in lower case.
```

Comment:
In Json Format?

Reply:
the first prompt at the beginning already specifies to return as a JSON so theres no need to repeat it